specification/netapp/resource-manager/Microsoft.NetApp/preview/2025-03-01-preview/netapp.json

```diff
@@ -7797,6 +7798,11 @@
```

Comment:
>Moving [](http://example.com/codeflow?start=21&length=6)

adding a Moving state, was this removed?

Reply:
I see this is intended as a bugfix also - it had been added in the other enum and references are getting redirected.